app/models/ingest_post_processor.rb

```diff
@@ -18,7 +18,9 @@ def customer_urn
   end
 
   def total_value
-    params.dig(:data, framework_definition.total_value_field)
+    value = params.dig(:data, framework_definition.total_value_field)
+    value = value.tr('$£, ', '').to_f if value.is_a?(String)
```

Comment:
I think a blacklist approach to bad characters leaves us at risk of running into unexpected characters at any point in the future, and this looks like it's coming straight from an external source. Are we confident that there will only ever be receiving `$`, `£` and ` `? 

I tried to test it with a euro character and got a result of `0.0` which seems they like something we ought to protect against or at least alert on.
```
irb(main):014:0> value = ' € 12,345.67  ' ; value.tr('$£, ', '').to_f if value.is_a?(String)
=> 0.0
```

Perhaps switching to a whitelist approach for allowing all currency symbols, alphanumerics and dots would help us with this?

Reply:
I was thinking about this morning actually. Might switch to an approach that strips anything that isn't a number or a period.